<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetGenericPropertiesFeet</class>
 <widget class="QWidget" name="WidgetGenericPropertiesFeet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="4" column="1">
    <widget class="QLineEdit" name="ModifiedDateInfield">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="DateCreatedInfield">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="ItemPathInfield">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="ItemLineInfield">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="ItemLineLabel">
     <property name="text">
      <string>Item line</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="InternalIdLabel">
     <property name="text">
      <string>UUID</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="HierarchyClusterInfield">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="ModifiedDateLabel">
     <property name="text">
      <string>Date modified</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="InternalInfield">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLineEdit" name="HierarchyPathInfield">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="ItemPathLabel">
     <property name="text">
      <string>Item path</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="HierarchyClusterLabel">
     <property name="text">
      <string>Hierarchy cluster</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="LinksLabel">
     <property name="text">
      <string>Links</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTableWidget" name="LinksTableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="HierarchyPathLabel">
     <property name="text">
      <string>Hierarchy path</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="CreatedDateLabel">
     <property name="text">
      <string>Date created</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="ItemClusterLabel">
     <property name="text">
      <string>Item cluster</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="ItemClusterInfield">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="AddLinkPushButton">
     <property name="toolTip">
      <string>Add a File or URL link</string>
     </property>
     <property name="text">
      <string>Add new link</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>LinksTableWidget</tabstop>
  <tabstop>AddLinkPushButton</tabstop>
  <tabstop>DateCreatedInfield</tabstop>
  <tabstop>ModifiedDateInfield</tabstop>
  <tabstop>InternalInfield</tabstop>
  <tabstop>ItemLineInfield</tabstop>
  <tabstop>ItemClusterInfield</tabstop>
  <tabstop>ItemPathInfield</tabstop>
  <tabstop>HierarchyClusterInfield</tabstop>
  <tabstop>HierarchyPathInfield</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
